cypress/support/auth.ts

```diff
@@ -48,6 +48,49 @@ Cypress.Commands.add('awxLogin', () => {
   cy.visit(`/`, { retryOnStatusCodeFailure: true, retryOnNetworkFailure: true });
 });
 
+Cypress.Commands.add('awxLoginTestUser', (username: string, password: string) => {
```

Comment:
Is this change related to work flow approvals? This command is not used as part of any test, right?

Reply:
This is related to workflow approvals, but I'll leave a comment.
Basically, when we have the access tab ready, some of the tests aim to:
admin assign user to approve workflow > logout > user logs in and approves